### PR TITLE
Bugfix 430 429 424

### PR DESCRIPTION
**Pull request by thegentlemanphysicist**
This tackles three tickets
- The lean bc link now works
- the products and services page is updated #431 
- the marketplace link has been added to the nav bar

Question: Should the link open in a new tab or not @HeatherRemacle @mark-a-wilson

### CODE REVIEW

File: react-frontend/src/components/Nav/navbar.js [hunk 1]
```diff
@@ -131,6 +133,15 @@ function NavBar() {
                 Products & Services
               </NavBarLink>
             </NavBarLi>
+            <NavBarLi>
```
Comment (mark-a-wilson) [inline react-frontend/src/components/Nav/navbar.js:136]: Would you like to add another menu item, before Marketplace, for Case Studies? Then I can blow up #443.

File: react-frontend/src/components/products/considering.js [hunk 2]
```diff
@@ -10,16 +10,16 @@ const cards = [
...
       <HrefLink
         href="mailto:exchangelab@gov.bc.ca"
         target="_blank"
         rel="noopener noreferrer"
       >
         Exchange Lab
       </HrefLink>
-      , a community of digital experts who are working together to address
-      urgent challenges.
+      ,a community of digital experts who are working together to address urgent
```
Comment (mark-a-wilson) [inline react-frontend/src/components/products/considering.js:21]: need a space between the comma and the a

File: react-frontend/src/components/Nav/navbar.js [hunk 3]
```diff
@@ -131,6 +133,15 @@ function NavBar() {
...
+              <NavBarLinkExternal href="https://digital.gov.bc.ca/marketplace">
+                Marketplace
+                <FontAwesomeIcon
+                  icon={faExternalLinkAlt}
+                  style={{ paddingLeft: '5px', height: 'inherit' }}
```
Comment (mark-a-wilson) [inline react-frontend/src/components/Nav/navbar.js:141]: Yes, I think the external link icon and opening in a new tab/window is good.

However, this is what I see when I run the branch:

Screen Shot 2020-11-23 at 11 07 03 AM


When I inspect the icon (svg), the `height: inherit` seems to be causing it. Maybe just remove the height property here?

Comment (thegentlemanphysicist) [reply react-frontend/src/components/Nav/navbar.js:141]: hmm, which browser?

File: react-frontend/src/components/Nav/navbar.js (hunk 1 above)
Comment (thegentlemanphysicist) [reply react-frontend/src/components/Nav/navbar.js:136]: sure

Comment (thegentlemanphysicist) [reply react-frontend/src/components/Nav/navbar.js:136]: done

File: react-frontend/src/components/Nav/navbar.js (hunk 3 above)
Comment (thegentlemanphysicist) [reply react-frontend/src/components/Nav/navbar.js]: should be fixed now

File: react-frontend/src/components/products/considering.js (hunk 2 above)
Comment (thegentlemanphysicist) [reply react-frontend/src/components/products/considering.js]: fixed

File: react-frontend/src/components/Nav/navbar.js (hunk 3 above)
Comment (mark-a-wilson) [reply react-frontend/src/components/Nav/navbar.js]: It was on Safari. All fixed now.

**Review by mark-a-wilson — APPROVED** [review]
Looks good - thanks for the fixes.

**Pull request merged**